demisto_sdk/commands/coverage_analyze/coverage_report.py

```diff
@@ -134,8 +134,8 @@ def coverage_diff_report(self) -> bool:
             min_cov = self.file_min_coverage(file_name)
             if min_cov > cov_precents:
                 logger.error(
-                    f"[red]Unit-tests for '{file_name}' must reach a coverage of at least {min_cov}% "
-                    f"(currently: {cov_precents}%).[/red]"
+                    f"[red]Unit-tests for '{file_name}' must reach a coverage of at least {min_cov} percents "
+                    f"(currently: {cov_precents} percents).[/red]"
```

Comment:
```suggestion
                    f"[red]Unit-tests for '{file_name}' must reach a coverage of at least {min_cov} percent "
                    f"(currently: {cov_precents} percent).[/red]"
```
but why this change?

Reply:
@ilaner and I made this change because there was a bug since we can't use % in the logger due to the format string.

Reply:
still should be percent not percents

Reply:
fixed